ganga/GangaTest/Lib/TestRemoteBackend/__init__.py

```diff
@@ -0,0 +1,19 @@
+
+import GangaCore.Utility.Config
+from .TestRemoteBackend import DummyRemote
+
+__all__ = ['DummyRemote']
+
+config = GangaCore.Utility.Config.makeConfig(
+    'TestDummyRemote', 'A dummy remote backend to emulate non local endpoints such as DIRAC')
+config.addOption("SERVER_PORT", 5100, "Port that the dummy server runs on")
+config.addOption("SERVER_DEFAULT_DELAY", 0.1, "Artificial delay in seconds")
+config.addOption("ENABLE_FINALISATION", True,
+                 "Whether or not to simulate job finalisation via dummy output file download.")
+config.addOption("FINALISATION_DELAY", 10, "Job finalisation simulation delay in seconds.")
+
+
+from .server import DummyServer
+
+server = DummyServer()
+server.start()
```

Comment:
This is what is causing the database suite to fail if you look carefully at the output. The chain of events is something like
- The database testing happens inside a dedicated container
- When a test inherits from `GangaUnitTest` it is triggering the loading of the `GangaTest` package.
- This in turn causes the `DummyServer` to be started.
- For some reason this fails inside the container we use for DB testing. In itself an unrelated error that I do not think is related to Ganga.

I think the solution would be to delay the start of the `DummyServer` until it is actually required (i.e. that a job with that backend is created). Art the moment ANY test that inherits from `GangaUnitTest` will cause the server to start.

Reply:
Right, I will move the server initialization and resubmit the changes.